koku/api/report/test/ocp/tests_views.py

```diff
@@ -218,6 +219,9 @@ def test_generic_report_ocp_cpu_success(self, mock_handler):
         )
 
         django_request = HttpRequest()
+        if not django_request.META.get('HTTP_HOST'):
+            django_request.META['HTTP_HOST'] = 'testhost'
```

Comment:
Broken Test: The Django test client doesn't set META params by default. But DRF depends on looking up the hostname in META. So, for the test to work, we need to set this ourselves.